libs/safe/src/lib/components/ui/alert/alert.stories.ts

```diff
@@ -1,31 +1,29 @@
 import { Meta, moduleMetadata, StoryFn } from '@storybook/angular';
 import { SafeAlertModule } from './alert.module';
 import { SafeAlertComponent } from './alert.component';
-import { AlertVariant } from './alert-variant.enum';
+import { SafeIconModule } from '../icon/icon.module';
 
 export default {
   component: SafeAlertComponent,
   decorators: [
     moduleMetadata({
-      imports: [SafeAlertModule],
+      imports: [SafeAlertModule, SafeIconModule],
     }),
   ],
   title: 'UI/Alert',
   argTypes: {
     variant: {
-      options: [
-        AlertVariant.DEFAULT,
-        AlertVariant.PRIMARY,
-        AlertVariant.SUCCESS,
-        AlertVariant.DANGER,
-        AlertVariant.WARNING,
-      ],
+      options: ['default', 'primary', 'success', 'danger', 'warning'],
```

Comment:
```suggestion
      options: variants,
```

And import it from your new alert-variant.ts, in case we change anything we will always keep track of the current types